composer.json

```diff
@@ -19,6 +19,7 @@
         "phpunit/phpunit": "^8.5",
         "symfony/framework-bundle": "^4.4.20 || ^5.2.5 || ^6.0",
         "nyholm/psr7": "^1.2",
+        "nyholm/symfony-bundle-test": "dev-master",
```

Comment:
The required branch is dev-master because, currently, the last available tag is not yet fully compatible with symfony 6. To be changed after a new tag is released.

Reply:
@ch3ric @aallier what do you think about using `dev-master`? I think it makes sense, considering it's only used for automated testing.

Reply:
If we want to get a chance for our fork to be merged in original repo, then it would be best to use a specific version, like it was before. But if there is no specific version fully compatible with symfony 6 then we don't have any other choice.

What issue do you get exactly with last release? I'm asking because from what I see in the changelog https://github.com/SymfonyTest/symfony-bundle-test/blob/master/CHANGELOG.md#181 version 1.8.1 is supposed to support symfony 8

Reply:
it seems that tag 1.8.1 is compatible with symfony 6: https://github.com/SymfonyTest/symfony-bundle-test/issues/65#issuecomment-1058111061 did you try?

Reply:
I just tried that and it only works with master:
```
➜  loco-adapter git:(EB-20725-CR) ✗ make phpunit   
PHP Fatal error:  Uncaught Error: Class "Nyholm\BundleTest\TestKernel" not found in /home/catalin/code/ets/docker/projects/loco-adapter/tests/Functional/BundleInitializationTest.php:19
Stack trace:
#0 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/src/Util/FileLoader.php(66): include_once()
#1 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/src/Util/FileLoader.php(54): PHPUnit\Util\FileLoader::load()
#2 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/src/Framework/TestSuite.php(383): PHPUnit\Util\FileLoader::checkAndLoad()
#3 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/src/Framework/TestSuite.php(481): PHPUnit\Framework\TestSuite->addTestFile()
#4 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/src/Util/Configuration.php(1031): PHPUnit\Framework\TestSuite->addTestFiles()
#5 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/src/Util/Configuration.php(900): PHPUnit\Util\Configuration->getTestSuite()
#6 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/src/TextUI/Command.php(963): PHPUnit\Util\Configuration->getTestSuiteConfiguration()
#7 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/src/TextUI/Command.php(202): PHPUnit\TextUI\Command->handleArguments()
#8 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/src/TextUI/Command.php(194): PHPUnit\TextUI\Command->run()
#9 /home/catalin/code/ets/docker/projects/loco-adapter/vendor/phpunit/phpunit/phpunit(98): PHPUnit\TextUI\Command::main()
#10 {main}
  thrown in /home/catalin/code/ets/docker/projects/loco-adapter/tests/Functional/BundleInitializationTest.php on line 19
make: *** [Makefile:16: phpunit] Error 255
```

Reply:
isn't this just because AppKernel was renamed to TestKernel in v2: https://github.com/SymfonyTest/symfony-bundle-test/blob/master/CHANGELOG.md#200 ? so in 1.8.X you need to use AppKernel in the test instead of TestKernel?

Reply:
I tried that, but it's still not compatible with Symfony 6.0:

```
PHP Fatal error:  Declaration of Nyholm\BundleTest\AppKernel::getProjectDir() must be compatible with Symfony\Component\HttpKernel\Kernel::getProjectDir(): string in /home/catalin/code/ets/docker/projects/loco-adapter/vendor/nyholm/symfony-bundle-test/src/AppKernel.php on line 94
```

Reply:
ok :/